tool/teleport/common/teleport.go

```diff
@@ -638,6 +644,49 @@ func onSCP(scpFlags *scp.Flags) (err error) {
 	return trace.Wrap(cmd.Execute(&StdReadWriter{}))
 }
 
+type compositeCh struct {
```

Comment:
Maybe move this to an sftp.go file?

Reply:
I can move it to a new file, I just thought since the amount of code added was so small it could stay in this file. Would you want it in the same package?

Reply:
Done